Singularity/Singularity/Screen/ScreenClasses/StatisticsScreen.cs

```diff
@@ -13,47 +16,100 @@ namespace Singularity.Screen.ScreenClasses
+        private SpriteFont mLibSans14;
 
-        public bool Loaded { get; set; }
+        // Buttons
+        private Button mCloseButton;
+
+        // Backup of StatisticsScreen position to update button positions.
+        private Vector2 mPrevPosition;
+
+        // ScreenManager to add / close screens.
+        private readonly IScreenManager mScreenManager;
+
+        // Director to gain access to StoryManager.
+        private readonly Director mDirector;
+
+
+        public StatisticsScreen(Vector2 screenSize, IScreenManager screenManager, Director director)
+        {
+            mScreenManager = screenManager;
+            mDirector = director;
+
+            Position = new Vector2(screenSize.X / 2 - 205, screenSize.Y / 2 - 225);
+            mPrevPosition = Position;
+
+        }
 
         /// <summary>
         /// Updates the contents of the screen.
         /// </summary>
         /// <param name="gametime">Current gametime. Used for actions
-        /// that take place over time.</param>
+        /// that take place over time </param>
         public void Update(GameTime gametime)
         {
-            throw new NotImplementedException();
+            mCloseButton.Update(gametime);
         }
 
         /// <summary>
-        /// Draws the content of this screen.
+        /// Loads any content that this screen might need.
         /// </summary>
-        /// <param name="spriteBatch">spriteBatch that this object should draw to.</param>
-        public void Draw(SpriteBatch spriteBatch)
+        /// <param name="content">Content Manager that should handle the content loading</param>
+        public void LoadContent(ContentManager content)
         {
-            throw new NotImplementedException();
+            mLibSans20 = content.Load<SpriteFont>("LibSans20");
+            mLibSans14 = content.Load<SpriteFont>("LibSans14");
+
+            mCloseButton = new Button("Close", mLibSans20, new Vector2(Position.X + 167, Position.Y + 410)) { Opacity = 1f };
+
+            mCloseButton.ButtonReleased += CloseButtonReleased;
         }
 
         /// <summary>
-        /// Loads any content that this screen might need.
+        /// Draws the content of this screen.
         /// </summary>
-        /// <param name="content">Content Manager that should handle the content loading</param>
-        public void LoadContent(ContentManager content)
+        /// <param name="spriteBatch">spriteBatch that this object should draw to.</param>
+        public void Draw(SpriteBatch spriteBatch)
         {
-            throw new NotImplementedException();
+            spriteBatch.Begin();
+
+            spriteBatch.FillRectangle(Position, new Vector2(400, 450), new Color(0.27f, 0.5f, 0.7f, 1f), 0f);
+            mCloseButton.Draw(spriteBatch);
+            // Draw the window name and stats names.
+            spriteBatch.DrawString(mLibSans20, "Statistics", new Vector2(Position.X + 145, Position.Y + 10), Color.Black);
```

Comment:
Please move all strings that are to be drawn into their own const fields. That way, if anything needs to be changed it's a quick fix and we don't have to track it down somewhere in the draw method.